content/docs/getting-started/installation/kubernetes/powerflex/csmoperator/_index.md

```diff
@@ -154,9 +154,12 @@ run `/opt/emc/scaleio/sdc/bin/drv_cfg --add_mdm --ip 10.xx.xx.xx.xx,10.xx.xx.xx`
    | ***Common parameters for node and controller*** |
    | X_CSI_VXFLEXOS_ENABLELISTVOLUMESNAPSHOT | Enable list volume operation to include snapshots (since creating a volume from a snap actually results in a new snap) | No | false |
    | X_CSI_VXFLEXOS_ENABLESNAPSHOTCGDELETE | Enable this to automatically delete all snapshots in a consistency group when a snap in the group is deleted | No | false |
-   | X_CSI_DEBUG | To enable debug mode | No | true |
+   | X_CSI_DEBUG | Enable/disable gocsi library level debugging. | No | false |
    | X_CSI_ALLOW_RWO_MULTI_POD_ACCESS | Setting allowRWOMultiPodAccess to "true" will allow multiple pods on the same node to access the same RWO volume. This behavior conflicts with the CSI specification version 1.3. NodePublishVolume description that requires an error to be returned in this case. However, some other CSI drivers support this behavior and some customers desire this behavior. Customers use this option at their own risk. | No | false |
    | INTERFACE_NAMES | A mapping of node names to interface names. Only necessary when SDC is disabled. | No | none |
+   | CSI_LOG_LEVEL | Sets the logging level of the driver. | No | INFO |
+   | GOSCALEIO_DEBUG | Enable/Disable goscaleio library-level debugging. | No | false |
+   | GOSCALEIO_SHOWHTTP | Enable/Disable goscaleio library-level REST request logging. Enabling will also **enable** GOSCALEIO_DEBUG regardless of GOSCALEIO_DEBUG setting. | No | false |
```

Comment:
For gocsi you use "library level" and here you use "library-level". These descriptions should be consistent.